webapp/components/results-navigation.html

```diff
@@ -51,6 +51,10 @@ <h2>Interoperability</h2>
             type: Number,
             computed: 'computeSelectedTab(tab)',
             value: 0,
+          },
+          query: {
+            type: String,
+            value: window.location.search,
```

Comment:
Does every "virtual navigation" (using history API) instantiate a new `results-navigation`? If not, this may need to be a binding instead.

Reply:
@mdittmer - path now updates when user 'navigates' inside the page.